### PR TITLE
Empty list placeholder -> Add placeholder to movie details

### LINKED ISSUE #90: Empty list placeholder
When the user doesn't have any data in the database sees an empty screen. Therefore I should add a due placeholder for this case.

### Note to subtask `Rename flag in view model`

After talking with @BorysChajdas I change the flag name `isEmptyDatabase` to `isPlaceholderVisible` because now it is more meaningful. It indicates what is stored, I use it to show or hide a placeholder if the database is empty. Using isEmptyDatabase is not a good way because, in a place where I use it, it means nothing.

- [x] Add placeholder to empty movie list
- [x] Use include and data binding
- [x] Rename flag in view model
- [x] Add placeholder to movie details

### PR DESCRIPTION
**Pull request by Nataniel-Antosik**

### CODE REVIEW

File: app/src/main/java/com/antosik/benchproject/app/movie/details/viewModel/MovieDetailsViewModel.kt [hunk 1]
```diff
@@ -24,10 +24,13 @@ class MovieDetailsViewModel @Inject constructor(
         get() = _isLoaderVisible
     val isScreenElementsVisible: LiveData<Boolean>
         get() = _isVisibleScreenElements
+    val isEmptyDatabase: LiveData<Boolean>
```
Comment (BorysChajdas) [inline app/src/main/java/com/antosik/benchproject/app/movie/details/viewModel/MovieDetailsViewModel.kt]: You rather should not use names like `Database` on UI layer (view, ViewModel) 😄. Use a more abstract name.

Comment (BorysChajdas) [reply app/src/main/java/com/antosik/benchproject/app/movie/details/viewModel/MovieDetailsViewModel.kt]: In this case, you probably should follow what you did in XML, so it should be just `isPlaceholderVisible`. You should try to name UI layer variables more "what it changes on the screen when set to true?" than "why it is set to true?".

**Review by BorysChajdas — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/antosik/benchproject/app/movie/details/viewModel/MovieDetailsViewModel.kt (hunk 1 above)
Comment (Nataniel-Antosik) [reply app/src/main/java/com/antosik/benchproject/app/movie/details/viewModel/MovieDetailsViewModel.kt]: Yes, it's my bad. I suggested the wrong older pull request (situation before fixing the name of this flag) I have the same name `isPlaceholderVisible` in another view model 😅 I will change it

Comment (Nataniel-Antosik) [reply app/src/main/java/com/antosik/benchproject/app/movie/details/viewModel/MovieDetailsViewModel.kt]: I changed it 😄

**Review by BorysChajdas — APPROVED** [review]

**Pull request merged**